Antidote/ActiveCallViewController.m

```diff
@@ -223,10 +239,20 @@ - (void)installConstraints
         make.top.equalTo(self.topViewContainer.topMargin);
     }];
 
-    [self.tableViewOfPausedCalls makeConstraints:^(MASConstraintMaker *make) {
+    [self.pauseCallsContainer makeConstraints:^(MASConstraintMaker *make) {
         make.top.equalTo(self.topViewContainer.bottom);
         make.width.equalTo(self.view);
-        make.bottom.equalTo(self.view.bottom).with.offset(-kTableViewBottomOffSet);
+        make.bottom.equalTo(self.view);
```

Comment:
That's because the tableView wouldn't know where to put the top at since it won't share a common superview with `self.topViewContainer`